control_parameters: # NOTE: This must match the node name in the cpp file
  ros__parameters:
    kp: 100.0
    # More to be added later
